tests/ArgumentTest.php

```diff
@@ -82,7 +82,8 @@ protected function getFullArguments()
     }
 
     /**
-     * @test
+     * @deprecated -- assertInternalType()
+     * @link https://github.com/sebastianbergmann/phpunit/issues/3369
```

Comment:
I've updated these tests in fe308a8f064f063f9d07c10ff1b2ea81dfcfc40c now so there's no need to disable them

Reply:
This is weird -- I thought I grabbed everything from master before I started this PR...